src/components/Facility/ConsultationDetails/ConsultationFeedTab.tsx

```diff
@@ -193,7 +194,7 @@ export const ConsultationFeedTab = (props: ConsultationTabProps) => {
                   feedStateSessionKey,
                   JSON.stringify({
                     type: "position",
-                    assetBed: preset.asset_bed.id,
+                    assetBed: "",
```

Comment:
if we are no longer going to cache asset bed id, we could remove that from the type and omit this right?

Reply:
Hmm, yea we could remove that.